src/constants/CardPropTypes.js

```diff
@@ -74,6 +74,7 @@ export const ValueCardPropTypes = {
   cardVariables: PropTypes.objectOf(
     PropTypes.oneOfType([PropTypes.string, PropTypes.func, PropTypes.number, PropTypes.bool])
   ),
+  customFormatter: PropTypes.func,
```

Comment:
Perhaps we should add a comment that explains this function and the parameters

Reply:
added a comment for this